lms/djangoapps/ci_lrs/tests.py

```diff
@@ -0,0 +1,48 @@
+import random
+from unittest.mock import patch, MagicMock
+
+from django.conf import settings
+from django.utils import timezone
+from django.test import TestCase, override_settings
+from pymongo.errors import ConnectionFailure, DuplicateKeyError, NetworkTimeout, OperationFailure, ServerSelectionTimeoutError
+
+from ci_lrs.utils import write_lrs_record_to_mongo
+
+
+class TestLRSTasks(TestCase):
+    def setUp(self):
+        self.mongodb = settings.LRS_INTEGRATIONS_MONGO_DB[settings.LRS_INTEGRATIONS_MONGO_COLLECTION]
```

Comment:
due to paranoia I wouldn't use settings in a test - I'd specify the mongo db and collection names as hard coded strings and call them stuff like "integrations_test_db" which can't possibly collide with production dbs that are accidentally open on a ssh tunnel

Reply:
Sure. Did that for the tests.